pkg/ccl/baseccl/encryption_spec.go

```diff
@@ -127,8 +128,7 @@ var _ pflag.Value = &StoreEncryptionSpecList{}
 func (encl StoreEncryptionSpecList) String() string {
 	var buffer bytes.Buffer
 	for _, ss := range encl.Specs {
-		// TODO(mberhault): use the FlagInfo once refactored.
-		fmt.Fprintf(&buffer, "--enterprise-encryption=%s ", ss)
+		fmt.Fprintf(&buffer, "--%s=%s ", cliflagsccl.EnterpriseEncryption.Name, ss)
```

Comment:
@tschottdorf: I think that's the line you wanted. So much work extra stuff for that one line.